src/scss/_move-popup.scss

```diff
@@ -5,7 +5,7 @@
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
 .move-popup-dialog {
-  height: 0;
+  height: fit-content;
```

Comment:
Hmm, so that's why that was there. I had recently changed this because Firefox flips out on it and makes the dialog super tall. Let me see if there's a compromise, or at least I'll document why this is here.